packages/fx-core/src/core/index.ts

```diff
@@ -543,11 +530,12 @@ class CoreImpl implements Core {
 
             const signinAzureCallback = async (validFxProject: boolean, args?: any[]): Promise<Result<null, FxError>> => {
                 this.ctx?.telemetryReporter?.sendTelemetryEvent(TelemetryEvent.LoginStart, {
-                    [TelemetryProperty.TriggerFrom]: args && args.toString() === "TreeView" ? TelemetryTiggerFrom.TreeView : TelemetryTiggerFrom.CommandPalette,
+                    [TelemetryProperty.TriggerFrom]: args && args[0].toString() === "TreeView" ? TelemetryTiggerFrom.TreeView : TelemetryTiggerFrom.CommandPalette,
```

Comment:
why this change?

Reply:
Because I need second parameter to indicate if need show dialog when sign in